src/ansys/dpf/gate/collection_grpcapi.py

```diff
@@ -246,7 +246,9 @@ def collection_get_support(collection, label):
         request.collection.CopyFrom(collection._internal_obj)
         if collection._server.meet_version("5.0"):
             request.label = label
-            request.type = base_pb2.Type.Value("SUPPORT")
+            request.type = base_pb2.Type.Value("SUPPORT")        
```

Comment:
@cbellot000 The changes should only be automatic, right?

Reply:
yes, I had a forgotten to copy this change in my pydpf gate PR. I have a PR there now, could you approve it please?